src/components/Votes/ApprovalCastVoteButton/ApprovalCastVoteButton.tsx

```diff
@@ -77,6 +87,58 @@ export default function ApprovalCastVoteButton({
   return (
     <VStack className="flex-shrink-0">
       <VStack alignItems="items-stretch">
+        {isReady && (
+          <div className="pt-3">
+            <TooltipProvider>
+              <Tooltip>
+                <TooltipTrigger className="w-full flex items-center justify-center gap-1 text-primary font-medium cursor-help">
+                  <span className="flex items-center">
+                    Proposal voting power{"\u00A0"}
+                    <TokenAmountDisplay amount={votingPower.totalVP} />
```

Comment:
Pedro was working on some changes to this component in [this PR](https://github.com/voteagora/agora-next/pull/757). Likely does not matter for getting this live quickly, but may introduce a merge conflict